doc/tutorials/core/basic_geometric_drawing/basic_geometric_drawing.markdown

```diff
@@ -120,95 +77,31 @@ Explanation
         -   The line thickness is set to **thickness** (in this case 2)
         -   The line is a 8-connected one (**lineType** = 8)
     -   *MyEllipse*
-        @code{.cpp}
-        void MyEllipse( Mat img, double angle )
-        {
-            int thickness = 2;
-            int lineType = 8;
-
-            ellipse( img,
-               Point( w/2.0, w/2.0 ),
-               Size( w/4.0, w/16.0 ),
-               angle,
-               0,
-               360,
-               Scalar( 255, 0, 0 ),
-               thickness,
-               lineType );
-        }
-        @endcode
+        @snippet cpp/tutorial_code/core/Matrix/Drawing_1.cpp myellipse
+
         From the code above, we can observe that the function @ref cv::ellipse draws an ellipse such
         that:
 
         -   The ellipse is displayed in the image **img**
-        -   The ellipse center is located in the point **(w/2.0, w/2.0)** and is enclosed in a box
-            of size **(w/4.0, w/16.0)**
+        -   The ellipse center is located in the point **(w/2, w/2)** and is enclosed in a box
+            of size **(w/4, w/16)**
```

Comment:
@StevenPuttemans , `ellipse` accepts `Point` and `Size` which are typedefs for [Point2i](http://docs.opencv.org/master/dc/d84/group__core__basic.html#ga1e83eafb2d26b3c93f09e8338bcab192) and [Size2i](http://docs.opencv.org/master/dc/d84/group__core__basic.html#ga346f563897249351a34549137c8532a0) respectively. So the tutorial code is correct - there should be no `.0`.